server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/executor/ClusterConfigTaskExecutor.java

```diff
@@ -362,6 +363,29 @@ public SettableFuture<ConfigTaskResult> loadConfiguration(boolean isCluster) {
     return future;
   }
 
+  @Override
+  public SettableFuture<ConfigTaskResult> setSystemStatus(boolean isCluster, NodeStatus status) {
```

Comment:
Is it necessary to use `isCluster` in `ClusterConfigTaskExecutor`?

Reply:
Rename isCluster to onCluster.